windows/winmme/JackWinMMEDriver.cpp

```diff
@@ -24,408 +24,405 @@ Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 #include "JackWinMMEDriver.h"
 #include "driver_interface.h"
 
-using Jack::JackWinMMEDriver;
```

Comment:
Is anything changed in this file except white space, indentation, line shifts, and so forth? If it is, it would be better if it could be done in separate commits, one for indentation/lineshifts/etc. changes, and one for code changes.

Reply:
I'll revert it, I commited on windows using powershell and notepad++, but with emacs on my linux I can clean stuff like this up. Thanks !